packages/console/src/components/Tooltip/index.module.scss

```diff
@@ -1,25 +1,52 @@
 @use '@/scss/underscore' as _;
 
-.container {
+.tooltip {
   position: absolute;
-  transform: translate(-50%, -100%);
   border-radius: 8px;
   background: #191c1d;
   color: #e0e3e3;
   box-shadow: var(--light-shadow-s1);
   padding: _.unit(2) _.unit(3);
   font: var(--font-body-medium);
 
-  &::after {
+  &.tooltipDefault {
+    background: #191c1d;
+    color: #e0e3e3;
+  }
+
+  &.tooltipSuccess {
+    background: #008a71;
+    color: #fff;
+  }
+
+  .arrow {
     content: '';
     display: block;
     position: absolute;
     border-width: 4px;
     border-style: solid;
     border-color: #191c1d transparent transparent;
     left: 50%;
-    top: 100%;
-    transform: translateX(-50%);
     box-shadow: var(--light-shadow-s1);
```

Comment:
I believe this `--light-shadow-s1` is deprecated and removed, and this one somehow failed to be found and replaced...
Please change it to `--shadow-1` instead, thanks.

Reply:
updated